### ISSUE TITLE
Accessibility feedback for Sitewide Feedback Button

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#26016]
# Accessibility feedback for Sitewide Feedback Button

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh and Angela

**Comment by joshkimux** [comment]
## Bugs

<img width="832" alt="Screen Shot 2021-06-14 at 3 27 57 PM" src="https://user-images.githubusercontent.com/14154792/121948359-27ca8580-cd25-11eb-8a61-a02011d0a390.png">

**Comment by joshkimux** [comment]
On zoomed layout, fields are hidden by close and submit buttons

<img width="1792" alt="Screen Shot 2021-06-14 at 3 29 14 PM" src="https://user-images.githubusercontent.com/14154792/121948471-4892db00-cd25-11eb-8e75-415da6084372.png">

**Comment by sshein** [comment]
oh, cool that you did this. Where are the fixes for this living, do we know @joshkimux ?

**Comment by joshkimux** [comment]
@sshein , I've included some guidance in the tickets which should be linked to this epic via zenhub. [Here's an example of one of the launchblockers.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/26017)

The recommended fix would be to place the feedback button at the bottom of the page above the footer in a fixed location, not sticky.
